1. Building a To-Do List Without Framework/submission/index.html

```diff
@@ -0,0 +1,128 @@
+                                    <h5
+                                        class="modal-title"
+                                        id="exampleModalLabel"
+                                    >
+                                        ADD A TODO LIST
+                                    </h5>
+                                    <button
+                                        type="button"
+                                        class="btn-close"
+                                        data-bs-dismiss="modal"
+                                        aria-label="Close"
+                                    ></button>
+                                </div>
+                                <form id="todo-form">
+                                    <div class="modal-body">
+                                        <div class="form-floating mb-3">
+                                            <input
+                                                type="text"
+                                                class="form-control"
+                                                id="floatingInput"
+                                                placeholder="Maths Homework"
+                                                name="todo-input"
+                                            />
+                                            <label for="floatingInput"
+                                                >Title</label
+                                            >
+                                        </div>
+                                    </div>
+                                    <div class="modal-footer">
+                                        <button
+                                            type="button"
+                                            class="btn btn-outline-danger fw-bold"
+                                            data-bs-dismiss="modal"
+                                        >
+                                            Close
+                                        </button>
+                                        <button
+                                            type="submit"
+                                            class="btn btn-outline-success fw-bold"
+                                        >
+                                            Save changes
+                                        </button>
+                                    </div>
+                                </form>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div id="confirmation" class="pt-5"></div>
+            <div id="todo-list"></div>
+
+            <div class="d-flex justify-content-center py-5">
+                <button
+                    class="btn btn-outline-success"
+                    id="save-todo"
+                    onclick="{saveTodo()}"
```

Comment:
@sauravkr818 good work on localStorage. However, there's a slight bug. If I delete a TODO and immediately reload the page, it persists on localStorage. This behavior may or may not be desirable. In such a case you could display a message to user prompting that they have unsaved changes and add an option to either proceed with changes or undo those changes.